src/env_config.py

```diff
@@ -35,6 +35,9 @@ def __init__(self):
         self.kitchen_simulator_interval = float(os.environ.get("KITCHEN_SIMULATOR_INTERVAL"))
         self.etl_interval = int(os.environ.get("ETL_INTERVAL"))
 
+        # flask config
+        self.secret_key = os.environ.get("SECRET_KEY")
```

Comment:
what is this secret key for? if it is needed why don't you name it `OAUTH2_SECRET_KEY`?

Reply:
done.